pontis-ui/src/components/common/Dots.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useId } from "react";
+
+interface DotsProps {
+  width: number;
+  height: number;
+  className: string;
+}
+
+const Dots: React.FC<DotsProps> = ({ width, height, className }) => {
```

Comment:
These are the dots behind the CTA at the bottom of each page. I have them as a component because I might use them elsewhere. If we think this is too fancy, we can also just remove them.

Reply:
no sounds good